olatundesleek/capcoinfx-fr
# #103 feature(user):implement profile update

#### What does this PR do?
Implement profile update
#### Description of Task to be completed?
- enable users to update their basic information
#### How should this be manually tested?
- clone the repository'
- cd into the project directory
- run `npm install`
- run `npm start` to start the development server
- login in with the following credentials
email: `admin@test.com`
password: `helloworld`
This will take you to the dashboard area if successfully authenticated
- once signed in, navigate to the `user profile` page to see the user's information
- update the values and hit the `save` button
#### Any background context you want to provide?
The local server runs at `http://localhost:3000`
#### What are the relevant pivotal tracker stories?
#103 
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
